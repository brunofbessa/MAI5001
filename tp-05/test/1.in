1
Fulano
1994
1
Fulana
1991
1
Zezinho
1999
1
Fulano
1999
0
