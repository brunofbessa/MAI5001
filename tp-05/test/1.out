Fulano 444.44
Fulana 111.11
Inexistente
Inexistente
